hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ECContainerReplicaCount.java

```diff
@@ -342,6 +342,10 @@ public List<Integer> overReplicatedIndexes(boolean includePendingDelete) {
     return indexes;
   }
 
```

Comment:
right now you are using it only for size? why it is returning set. Is there any other plans to use? Just healthy index count may work?